staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/validation/validation.go

```diff
@@ -82,6 +83,9 @@ func ValidateAPIService(apiService *apiregistration.APIService) field.ErrorList
 	if len(apiService.Spec.Service.Name) == 0 {
 		allErrs = append(allErrs, field.Required(field.NewPath("spec", "service", "name"), ""))
 	}
+	if errs := utilvalidation.IsValidPortNum(int(apiService.Spec.Service.Port)); errs != nil {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("spec", "service", "port"), apiService.Spec.Service.Port, "port is not valid:"+strings.Join(errs, ",")))
```

Comment:
nit: add spaces to message construction, `"port is not valid: "+strings.Join(errs, ", ")` (applies to all places this message was constructed)